DESCRIPTION

```diff
@@ -60,3 +60,5 @@ Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.2
 SystemRequirements: C++11
+Remotes:
+    r-lib/testthat
```

Comment:
There was a testthat issue that was causing some warnings when compiling cpp11test on Windows with the "latest" version of R. Since we use `options(warn = 2)` before compiling it was causing the build to fail https://github.com/r-lib/testthat/issues/1672

Corresponds to this commit https://github.com/r-lib/cpp11/pull/279/commits/7a479d84317fed184e8ee24c74cfbeaa4c5b356c

Reply:
wonder if we should pin this to a commit SHA rather than having it float, and remove the pin after the changes are on CRAN.

I worry that having it float with the HEAD would cause builds to fail due to unrelated testthat development